README.md

```diff
@@ -1,6 +1,64 @@
-# edgex-endorsement
+# EdgeX Endorsement
 
 ## Introduction
 
-Code, tools, and documentation for validating and endorsing device profiles written by third parties.
+Code, tools, and documentation for validating and endorsing device profiles written by third parties. At a high level you'll need to complete five steps to be eligible for the EdgeX Endorsement Program which will ensure that data provided meets all required components. 
+
+1) Deploy EdgeX using one of the provided cloud templates. Alternatively, you can leverage the latest released compose-file provided here: This ensures a vanilla version of EdgeX is being used without modification to ensure the highest compatibility.
+2)  Compose and deploy your device profile using the Device Profile Modeling tool provided [here](https://www.edgexfoundry.org/tbd). This will ensure the profile is validated and has all required components.
```

Comment:
1. This second-step should be the first step.

2. It is my understanding that we do not support nor provide tools for composing device profiles. Are we revisiting that decision?  Furthermore, it might be unwise to ask the user to now start composing their device profile (presumably a time-consuming affair) just after you had them start the meter on an EdgeX instance!

3. It my mind, the first step is validating the device profile, which does not require an EdgeX instance and hence, no  need to deploy the profile.  I am aware, though, that some suggestions include improving upon the built-in validation codebase in Core and utilizing that as part of the EdgeX Endorsement process, so we can start a seperate thread on device profile validation implementation.

I envision two user types and casually refer to them as low-torque/high-revolutions and high-torque/low-revolutions.  The former are those who are starting with a nearly blank sheet approach and iterate frequently as they build their device profile.  The latter have a nearly complete device profile in hand and need only a few iterations at most to fix the profile.

Reply:
I think the plan is to leverage Iotech device profile builder, building one by hand is probably the worst thing we could ask anyone to do and most certainly would be a time consuming affair.